src/renderer/hooks/useOutputDataStore.ts

```diff
@@ -57,6 +58,9 @@ export const useOutputDataStore = () => {
             },
             [setMap]
         ),
+        clearAll: useCallback(() => {
```

Comment:
The map operation is called `clear`.
```suggestion
        clear: useCallback(() => {
```